.idea/modules.xml

```diff
@@ -1,8 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
As I know it's user specific thing. And it is generating while you import gradle project to your IDE. Am I right?